addons/web_editor/static/tests/field_html_tests.js

```diff
@@ -132,6 +150,24 @@ QUnit.module('web_editor', {}, function () {
             form.destroy();
         });
 
+        QUnit.test('notebooks defined inside HTML field widgets are ignored when calling setLocalState', async function (assert) {
+            assert.expect(0);
+
+            var form = await testUtils.createView({
+                View: FormView,
+                model: 'note.note',
+                data: this.data,
+                arch: '<form>' +
+                    '<field name="body" widget="html" style="height: 100px"/>' +
+                    '</form>',
+                res_id: 2,
+            });
+            // just check that there is no error on clicking Edit
+            await testUtils.form.clickEdit(form);
+            await testUtils.nextTick();
```

Comment:
could you at least add an assert to check that the view is in edition (we don't like tests with expect(0))